web/tasks/Wizards of Wikipedia.md

```diff
@@ -10,110 +10,168 @@ motivation: TODO.
+
+Thr original repository has more [details](https://github.com/facebookresearch/ParlAI/blob/master/parlai/tasks/wizard_of_wikipedia/worlds.py)
 
 ### Data Statistics
 
-[More Information Needed]
+Some statistics for train split:
+- Number of Utterances : 166,787
+- Number of Dialogues: 18,430 
+- Number of Topics: 1,247
+- Average Turns per Dialogue: 9.0
 
 ## Dataset Creation
 
 ### Curation Rationale
 
-[More Information Needed]
+As mentioned in the Wizard-of-Wikipedia paper, goal of collecting data of wizard-apprentice conversations between humans is to then be able to train models which can replace the human wizard and speak to a human apprentice, similar to the procedure in Wizard of Oz experiments.
 
 ### Communicative Goal
 
-[More Information Needed]
+Apprentice talks to the wizard freely, playing the role of a curious learner, eager to chat.
+Human Wizard is asked discuss a topic with an eager apprentice.
 
 
 ### Source Data
 
+A set of 1365 natural, open-domain dialogue topics, each linked to a Wikipedia article, was crowd-sourced. These dialogues are on diverse topics such as commuting, Gouda cheese, music festivals, podcasts, and bowling.
+
+At each step of the dialogue the wizard has access to a set of passages of knowledge which may be relevant to the given dialogue context. The top 7 articles (first paragraph only) for the last two turns of dialogue (by wizard and apprentice) and the article (first 10 sentences only) for the original topic, and present these articles to the wizard as knowledge context, along with their titles. 
+
 #### Initial Data Collection and Normalization
 
-[More Information Needed]
+The final dialogue dataset we collect consists of 22,311 dialogues with
+201,999 turns, which we divide into 166,787 for train, 17,715 for validation, and 17,497 for test. The test set is split into two subsets, Test Seen and Test Unseen. Test Seen contains 533 overlapping topics with the training set with new dialogues about those topics. Test Unseen consists of 58 topics never seen before in train or validation. 
 
-#### Who are the source language producers?
 
-[More Information Needed]
 
 ### Annotations
 
 #### Annotation process
 
-[More Information Needed]
+Authors stated the following annotation procedure in their paper:
+1. Either the wizard or apprentice is picked to choose the topic and speak first. The other
+player receives the topic information, and the conversation begins.
+2. When the apprentice sends the wizard a message, the wizard is shown relevant knowledge
+(described below), and chooses a relevant sentence in order to construct a response, or else
+chooses the no sentence used option.
+3. The Wizard responds to the apprentice basing their response on their chosen sentence.
+4. The conversation repeats until one of the conversation partners ends the chat (after a minimum of 4 or 5 turns each, randomly chosen beforehand).
 
 #### Who are the annotators?
 
-[More Information Needed]
+Crowdworkers
```

Comment:
Can you add more here? See: https://gem-benchmark.github.io/tasks/guide#who-are-the-annotators 

It is fine to state that information is not available.